### ISSUE TITLE
1.16.1 Issue: Cannot find manifest for f9e737b7-f0df-4597-ba8c-3060f82380db_1.16.1. Store had [1.16.0]. Service worker version: (none),DurableN1=true

### ISSUE THREAD

**Issue by kbeeveer46** [SharePoint/sp-dev-docs#8623]
### Target SharePoint environment

SharePoint Online

### What SharePoint development model, framework, SDK or API is this about?

💥 SharePoint Framework

### Developer environment

Windows

### What browser(s) / client(s) have you tested

- [ ] 💥 Internet Explorer
- [X] 💥 Microsoft Edge
- [X] 💥 Google Chrome
- [ ] 💥 FireFox
- [ ] 💥 Safari
- [ ] mobile (iOS/iPadOS)
- [ ] mobile (Android)
- [ ] not applicable
- [ ] other (enter in the "Additional environment details" area below)

### Additional environment details

- SPFx version: 1.16.1
- Node.js version: 16.16.0

### Describe the bug / error

When I run my project and add a web part to the page I get the following error. I'm not sure where it's coming from or where to look to fix it. The web part was working fine this morning and the only thing I've done is updated my dependencies to 1.16.1. Is it possible one of my other dependencies needs to be updated to work with 1.16.1 like SPFx Fast Serve? I don't have any issues building the project. Only adding a web part to the page. The error goes away when I revert to 1.16.0.

```
Something went wrong

If the problem persists, contact the site administrator and give them the information in Technical Details.
TECHNICAL DETAILS

ERROR:
Cannot find manifest for f9e737b7-f0df-4597-ba8c-3060f82380db_1.16.1. Store had [1.16.0]. Service worker version: (none),DurableN1=true

CALL STACK:
Error: Cannot find manifest for f9e737b7-f0df-4597-ba8c-3060f82380db_1.16.1. Store had [1.16.0]. Service worker version: (none),DurableN1=true
    at e.getManifest (https://res-1.cdn.office.net/files/sp-client/sp-webpart-workbench-assembly_en-us_7c37646a7f4b0e091d2a805c37da5d1a.js:70:826262)
    at e.requestManifest (https://res-1.cdn.office.net/files/sp-client/sp-webpart-workbench-assembly_en-us_7c37646a7f4b0e091d2a805c37da5d1a.js:70:828047)
    at e._configureComponentResource (https://res-1.cdn.office.net/files/sp-client/sp-webpart-workbench-assembly_en-us_7c37646a7f4b0e091d2a805c37da5d1a.js:70:838409)
    at e.configLoadComponent (https://res-1.cdn.office.net/files/sp-client/sp-webpart-workbench-assembly_en-us_7c37646a7f4b0e091d2a805c37da5d1a.js:70:836624)
    at e._loadComponent (https://res-1.cdn.office.net/files/sp-client/sp-webpart-workbench-assembly_en-us_7c37646a7f4b0e091d2a805c37da5d1a.js:70:841017)
    at e.loadComponent (https://res-1.cdn.office.net/files/sp-client/sp-webpart-workbench-assembly_en-us_7c37646a7f4b0e091d2a805c37da5d1a.js:70:856680)
    at https://res-1.cdn.office.net/files/sp-client/sp-webpart-workbench-assembly_en-us_7c37646a7f4b0e091d2a805c37da5d1a.js:70:839820
    at new Promise (<anonymous>)
    at e.loadComponent (https://res-1.cdn.office.net/files/sp-client/sp-webpart-workbench-assembly_en-us_7c37646a7f4b0e091d2a805c37da5d1a.js:70:839706)
    at e.loadComponent (https://res-1.cdn.office.net/files/sp-client/sp-webpart-workbench-assembly_en-us_7c37646a7f4b0e091d2a805c37da5d1a.js:70:833078)
```

Dependencies

```
 "dependencies": {
        "@fluentui/react": "^8.103.3",
        "@microsoft/sp-core-library": "^1.16.1",
        "@microsoft/sp-lodash-subset": "^1.16.1",
        "@microsoft/sp-office-ui-fabric-core": "^1.16.1",
        "@microsoft/sp-property-pane": "^1.16.1",
        "@microsoft/sp-webpart-base": "^1.16.1",
        "@pnp/graph": "^3.9.0",
        "@pnp/sp": "^3.9.0",
        "@turf/turf": "6.5.0",
        "exceljs": "4.3.0",
        "file-saver": "2.0.5",
        "html2canvas": "1.4.1",
        "jspdf": "2.5.1",
        "jszip": "3.10.1",
        "primeicons": "6.0.1",
        "primereact": "^8.7.2",
        "react": "17.0.1",
        "react-auto-translate": "0.0.6",
        "react-dom": "17.0.1",
        "react-hook-size": "1.3.5",
        "react-json-editor-ajrm": "2.5.13",
        "react-map-gl": "6.1.17",
        "react-to-print": "^2.14.10",
        "react-transition-group": "4.4.5",
        "reactour": "1.18.7",
        "recharts": "^2.1.16",
        "styled-components": "5.3.6",
        "use-state-with-callback": "3.0.2"
    },
    "devDependencies": {
        "@microsoft/eslint-config-spfx": "^1.16.1",
        "@microsoft/eslint-plugin-spfx": "^1.16.1",
        "@microsoft/rush-stack-compiler-4.5": "0.3.0",
        "@microsoft/sp-build-web": "^1.16.1",
        "@microsoft/sp-module-interfaces": "^1.16.1",
        "@rushstack/eslint-config": "3.1.1",
        "@types/file-saver": "2.0.5",
        "@types/react": "17.0.45",
        "@types/react-dom": "17.0.17",
        "@types/react-json-editor-ajrm": "2.5.3",
        "@types/reactour": "1.18.2",
        "@types/recharts": "^1.8.24",
        "@types/webpack-env": "1.18.0",
        "ajv": "^8.11.2",
        "eslint": "^8.28.0",
        "eslint-plugin-react-hooks": "4.6.0",
        "gulp": "4.0.2",
        "spfx-fast-serve-helpers": "^1.16.0-beta.0",
        "typescript": "^4.9.3"
    }
```

Dev tools console window:

![image](https://user-images.githubusercontent.com/12972554/204879762-5c41264b-6af8-49c3-9517-502a907dd47d.png)

### Steps to reproduce

1. Upgrade dependencies to 1.16.1
2. Run project via SPFx Fast Serve
3. Add web part to page

### Expected behavior

I was expecting my web part to load and display on the page like it does when using 1.16.0

**Comment by omarelanis** [comment]
I am experiencing this exact same issue with a new project I just built, both when trying on the workbench and when running directly from within a page

**Comment by prateeknain** [comment]
I am also experiencing the exact same issue from 30 Dec 2022

**Comment by chr-sad** [comment]
I am also experiencing the same issue with and without removing the complete local cache.

**Comment by nbelyh** [comment]
Same here

**Comment by heinrich-ulbricht** [comment]
Same here. Upgraded a project from SPFx 1.15.2 to 1.16.1 - now the web part shows OP's error.

**Comment by StevenDerveaux-DnDCore** [comment]
Same here! Just start a blank new webpart project...

**Comment by ashokghoghari** [comment]
Getting same issue. Upgraded from SPFx 1.15.2 to 1.16.1

**Comment by Brodyagos** [comment]
Confirming having a similar problem after update to 1.16.1

**Comment by iClarkSi** [comment]
I had the same issue creating a SPFx 1.16.1 from scratch. The solution @kbeeveer46 did worked for me too. 

Just installing or changing SPFx version 1.16.0 works fine.

Thanks for the post!

**Comment by VesaJuvonen** [comment]
Thanks everyone for the input here - we are working on the root cause analyses and will come back on the details here asap.

**Comment by AJIXuMuK** [comment]
Just to provide the current status and expectations.
The issue is related to 1.16.1, not 1.16.0.
It doesn't matter if you upgrade an existing solution or create a new one using 1.16.1.
The problem is in internal service code.
The fix has been done and should be rolled out to 100% around December 12.
Until then we don't recommend updating to 1.16.1.

Sorry for the inconvenience.

**Comment by joywin-dev** [comment]
What is the cmd to move to the previous version? Do i need to uninstall the current version? please help

**Comment by Brodyagos** [comment]
> Просто для предоставления текущего состояния и ожиданий. Проблема связана с 1.16.1, а не с 1.16.0. Неважно, обновите ли вы существующее решение или создадите новое, используя 1.16.1. Проблема во внутреннем сервисном коде. Исправление было сделано и должно быть развернуто до 100% примерно 12 декабря. До тех пор мы не рекомендуем обновляться до 1.16.1.
> 
> Извините за беспокойство.

Спасибо за ответ, проблему понял, сам откатил версию. PS Ответил, чтобы привлечь внимание к проблеме

**Comment by Brodyagos** [comment]
> What is the cmd to move to the previous version? Do i need to uninstall the current version? please help

The easiest option is to do a web part search and replace all versions 1.16.1 with 1.16.0 (npmId, manifest, package.json)

**Comment by greggdavenport** [comment]
> > What is the cmd to move to the previous version? Do i need to uninstall the current version? please help
> 
> The easiest option is to search the web part and replace all versions 1.16.1 with 1.16.0 (npmId, manifest, package.json). And then remove the .lock file, reinstall the packages and build the solution again

This worked for me

**Comment by HiltonGiesenow** [comment]
> Just to provide the current status and expectations. The issue is related to 1.16.1, not 1.16.0. It doesn't matter if you upgrade an existing solution or create a new one using 1.16.1. The problem is in internal service code. The fix has been done and should be rolled out to 100% around December 12. Until then we don't recommend updating to 1.16.1.
> 
> Sorry for the inconvenience.

Argh! I've already delayed upgrading to 1.16.x because the guidance last week was to wait for 1.16.1 :-(. I'm patching a project that's dependent on sp-dev-fx-property-controls, but it's now taken a dependency on 1.16.x in the latest release

**Comment by VesaJuvonen** [comment]
Elaborating the situation, a bit for full transparency and how you can work with SPFx without concerns currently.

✅ Default for the SPFx package in npm has been updated back to point to the 1.16 version - so that by default you'll get the 1.16 version installed when you run installation from npm
✅ 1.16 is safe to use, but had few small issues which is why 1.16.1 was created - these were not critical, but inconvenience - like the sppkg file size was significantly larger vs the preview 1.15.2 version
✅ When the needed server-side fixes are rolling out worldwide for 1.16.1, we will automatically update the default installation version from npm point to the 1.16.1 version

This model will mean that everyone can install the default version from the npm without issues - and we'll update things automatically when the needed fixes are out. I'm also working on updating the release notes and blog posts with a temporary guidance.

Huge thanks for all on reporting the issues in this issue list - as that always helps on speeding up the root cause analyzes and resolution 👏

We do apologize the inconvenience and potential additional work caused with the issues in the 1.16.1 version. We are working on internal post-mortem to reduce the probability of similar issues happening in future 🤞

**Comment by AJIXuMuK** [comment]
The fix has been rolled out to 100% Prod.
We're back to 1.16.1 as the latest version on NPM and it is safe to use this version.
